app/javascript/components/rooms/RoomCard.jsx

```diff
@@ -15,15 +15,15 @@ export default function RoomCard(props) {
   const { handleStartMeeting, isLoading: startMeetingIsLoading } = useStartMeeting(friendlyId);
 
   return (
-    <Card className="rooms-card" border="light">
-      <Card.Body className="room-card-top pb-0" onClick={handleClick}>
+    <Card className="rooms-card" style={{ height: '19rem', overflow: 'hidden' }} border="light">
```

Comment:
I'm not sure inline styles is the way to go here. I'd rather some other pattern

Reply:
Sure, if you approve the changes it's ready to get changed into a seperate css file.